src/less/bundles/skin/ds4/skin.less

```diff
@@ -1,4 +1,5 @@
 // Source file for generated skin.min.css (used by website and CDN)
+@import "../../../progress-bar/ds4/progress-bar.less";
```

Comment:
This entry should go just before progress-stepper. Same change also needed in ds6 file below.